fuzzy-message-detection/src/fmd2.rs

```diff
@@ -11,15 +12,15 @@ use crate::{CcaSecure, FmdScheme, RestrictedRateSet};
 
 
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Serialize, Deserialize)]
```

Comment:
serde functionality should be gated under a feature flag, IMO
```suggestion
#[derive(Debug, Clone)]
#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
```